src/huggingface_hub/hf_api.py

```diff
@@ -180,7 +155,7 @@ def __repr__(self):
 
 class ModelInfo:
     """
-    Info about a public model accessible from huggingface.co
+    Info about a model accessible from huggingface.co
```

Comment:
Since we're touching this class, I think it'd be nice to add the info about init args here.